proxy/ParentSelection.h

```diff
@@ -248,18 +256,31 @@ struct ParentResult {
     }
   }
 
+  void
+  print()
+  {
+    printf("ParentResult - hostname: %s, port: %d, retry: %s, line_number: %d, last_parent: %d, start_parent: %d, wrap_around: %s, "
+           "last_lookup: %d, result: %s\n",
+           hostname, port, (retry) ? "true" : "false", line_number, last_parent, start_parent, (wrap_around) ? "true" : "false",
```

Comment:
@a-canary I just wanted to log the ParentResult struct values in unit tests.  It's not being used to identify yaml rules.  The ParentResult struct is also used in ParentSelection where line_number is used there to identify the parent.config line.  This will go away eventually but this print() function was only used in the unit test logs for debugging purposes and is not used by the NextHop selection .